src/IssueViz.Security.UnitTests/HotspotsList/HotspotTableEntryTests.cs

```diff
@@ -187,32 +187,15 @@ public void TryCreateColumnContent_UnsupportedColumn_ReturnsNull(string column)
         }
 
         [TestMethod]
-        public void TryCreateColumnContent_IssueVizIsNavigable_ReturnsNull()
```

Comment:
This test is not relevant anymore: a framework element is always created now with a context menu, and it either has navigability style or no style. I added the equivalent of this test in the WpfElementFactory tests